pkg/assembler/graphql/helpers/package.go

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"github.com/guacsec/guac/pkg/assembler/graphql/model"
+)
+
+// Return a package structure containing only IDs
+func GetPackageAsIds(packages []*model.Package) []*model.PackageIDs {
+	results := []*model.PackageIDs{}
+	for _, pkg := range packages {
+		resultPackage := model.PackageIDs{PackageTypeID: pkg.ID}
+		for _, namespace := range pkg.Namespaces {
+			resultNamespace := resultPackage
```

Comment:
nit: why create new variables here? Just keep using `resultPackage`

Reply:
I was writing a generic function to flatten the tree structure, hence the new variables to handle multiple namespaces, names,  versions.  I'll add a test case to highlight/validate the behaviour, this may make it more obvious.

Reply:
ah ok

Reply:
Test case has been added and pushed up